CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Added
```

Comment:
nit: Need a newline.

Also, we do not need changelog entires for the integration tests/demos/github workflows as well I think, since they are not builder facing.

Reply:
Added a newline. I do not think this needs a changelog entry based on your reason so I will just remove it.